docs-source/content/faq/cannot-pull-image.md

```diff
@@ -0,0 +1,151 @@
+  a Docker registry that is accessible the to worker nodes
+
+Let's review what happens when Kubernetes starts a pod.
+
+![Pulling an image](/weblogic-kubernetes-operator/images/image-pull.png)
+
+The definition of the pod contains a list of container specifications.  Each container 
+specificiation contains the name (and optionally tag) of the image that should be used
+to run that container.  In the example above, there is a container called `c1` which is
+configured to use the docker image `some.registry.com/owner/domain1:1.0`.  This image
+name is in the format `registry address / owner / name : tag`, so in this case the 
+registry is `some.registry.com`, then owner is `owner`, the image name is `domain` 
+and the tag is `1.0`.  Tags are a lot like version numbers, but they are not required 
+to be numbers or to be in any particular sequence or format.  If you omit the tag, it 
+is assumed to be `latest`.
+
+{{% notice note %}}
+The docker tag `latest` is confusing - it does not actually mean the latest version of 
+the image that was created or published in the registry, it just literally means 
+whichever version the owner decided to call "latest".  Docker and Kubernetes make 
+some assumptions about latest, and it generally recommended to avoid using it and instead
+specify the actual version/tag that you really want. 
+{{% /notice %}}
+
+First, Kuberentes will check to see if the requested image is available in the local 
+Docker image store on whichever worker node the pod was scheduled on.  If it is there,
+then it will use that image to start the container.  If it is not there, then Kubernetes
+will attempt to pull the image from a remote Docker registry.
+
+{{% notice note %}}
+There is another setting called `imagePullPolicy` that can be used to force Kubernetes 
+to always pull the image, even if it is already present in the local Docker image 
+store.
+{{% /notice %}}
+
+If the image is available in the remote registry and it is public, that is it does not 
+require authentication, then Kubernetes will pull the image to the local Docker image
+store and start the container. 
+
+#### Images that require authentication
+
+If the remote Docker registry requires authentication, then you will need to provide
+the authentication details in a Kubernetes `docker-registry` secret and tell Kubernetes
+to use that secret when pulling the image. 
+
+To create a secret, you can use the following command:
+
+```
+kubectl create secret docker-registry secret1 \
+        --docker-server=some.registry.com \
+        --docker-username=bob \
+        --docker-password=bigSecret \
+        --docker-email=bob@some.com \
+        --namespace=default
+```
+
+In this command, you would replace `secret1` with the name of the secret; the `docker-server`
+is set to the registry name, without the `https://` prefix; the `docker-username`, `docker-password`
+and `docker-email` are set to match the credentials you use the authenticate to the remote 
```

Comment:
credentials you use the authenticate to-> credentials you use to authenticate to